fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/exception/ConstraintViolationException.java

```diff
@@ -23,6 +23,7 @@
  * @author whikloj
  * @since 2015-05-29
  */
+@SuppressWarnings("WeakerAccess")
```

Comment:
This actually gives me a warning in eclipse:
`Unsupported @SuppressWarnings("WeakerAccess")`